frontend/components/InlineFeedbackPopover/Comment.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react'
+import {
+  useUpdateCommentMutation,
+  useDeleteCommentMutation,
+  CommentFragmentFragment as CommentType,
+} from '../../generated/graphql'
+import Button, { ButtonSize, ButtonVariant } from '../../elements/Button'
+import BlankAvatarIcon from '../Icons/BlankAvatarIcon'
+import theme from '../../theme'
+
+type CommentProps = {
+  comment: CommentType
+  canEdit: boolean
+  onUpdateComment: any
+}
+
+const Comment: React.FC<CommentProps> = ({ comment, canEdit, onUpdateComment }) => {
+  const [isEditMode, setIsEditMode] = React.useState<boolean>(false)
+  const [updatingCommentBody, setUpdatingCommentBody] = useState<string>(comment.body)
```

Comment:
nit: maybe `updatedCommentBody`? It does say `useState<string>`, but I had to double check that it wasn't a boolean

Reply:
Hmm only problem is that it's not just a 'past' thing, since this also represents the state when you begin editing, etc. At least we have the nice TS annotation so you can see it's a string! 😅 How about I add a comment above?

Reply:
Oh it's ok, I don't think a comment is necessary 😊  traditionally anything with `-ing` is a boolean, but it's really minor, so that's why i put `nit: ` 🤣 something i've used on other teams for nitpicky suggestions that you can feel free to ignore 😛